src/wipes/cinematic-in.css

```
@@ -0,0 +1,15 @@
+@keyframes wipe-cinematic-in {
+    0% {
+      clip-path: inset(100% 0 100% 0);
+    }
+    30%, 70% {
+      clip-path: inset(10% 0 10% 0);
+    }
+    100% {
+      clip-path: inset(0 0 0 0);
+    }
+  }
+
+[transition-style="cinematic:wipe:in"] {
```

```suggestion
[transition-style="in:wipe:cinematic"] {
```

this matches the current naming pattern 👍🏻